arangod/Cluster/ActionBase.h

```diff
@@ -112,6 +112,14 @@ class ActionBase {
   /// @brief finalize statistics
   void endStats();
 
+  /**
+   *  @brief    update progress by long running processes, while allowing to
+   *            abort the process by user.
+   *  @param  d percentage of work done
+   *  @return   abort if !ok(), true if ok(), with reason to abort.
```

Comment:
I do not understand this comment and thus I do not understand the API.

Reply:
Removed the conflicting part.